static/sass/main.scss

```diff
@@ -14,3 +14,9 @@ td {
 is-dark {
   background-color: $theme-default-nav;
 }
+
+.is-last {
```

Comment:
This is fine but might be improved with the selector that works forever:
```css
li.p-matrix__item:last-of-type:nth-of-type(3n + 1) {
```
Maybe propose this to Vanilla?

Reply:
Proposed here https://github.com/canonical/vanilla-framework/issues/4828